build.gradle

```diff
@@ -1,12 +1,16 @@
 plugins {
   id "com.github.hierynomus.license" version "0.14.0"
   id "com.jfrog.bintray" version "1.7.3"
+  // publish via m2 settings?
+  id 'maven-publish'
+  id 'net.linguica.maven-settings' version '0.5'
+  id "org.hibernate.build.maven-repo-auth" version "3.0.2"
 }
 
 allprojects {
 //  apply plugin: 'license'
 
-  project.version = "git describe --always --dirty".execute().text.trim()
+  project.version = "git describe --always --dirty".execute().text.trim() + "-SNAPSHOT"
```

Comment:
will need to update the version for release version. but we don't care for now. each different code build gives unique version anyway like `0.3.0-2-gb3e162c-SNAPSHOT`